website/docs/r/cse_rule_tuning_expression.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: "sumologic"
+page_title: "SumoLogic: sumologic_cse_rule_tuning_expression"
+description: |-
+  Provides a CSE Rule Tuning Expression
+---
+
+# rule_tuning_expression
+Provides a CSE Rule Tuning Expression.
+
+## Example Usage
+```hcl
+resource "sumologic_cse_rule_tuning_expression" "rule_tuning_expression" {
+  name = "New Rule Tuning Name"
+  description = "New Rule Tuning Description"
+  expression = "accountId = 1234"
+  enabled = "true"
+  exclude = "true"
+  is_global = "false"
+  rule_ids = ["LEGACY-S00084"]
+}
+```
+
+## Argument reference
+
+The following arguments are supported:
+
+- `name` - (Required) The name of the insights status.
+- `description` - (Required) The description of the insights status.
+- `expression` - (Required) Expression to match.
+- `enabled` - (Required) Enabled flag.
+- `exclude` - (Required) Set to true to exclude records that also match expression.
```

Comment:
Maybe something like this just to be extra clear here:
"Set to true to exclude records that match the expression. If set to false, only records that do match the expression will be included."

Reply:
done under:
https://github.com/SumoLogic/terraform-provider-sumologic/pull/281/commits/9b311156b3abd89688cd6761ba81798e3df37d59